app/views/pair_requests/_form.html.erb

```diff
@@ -1,32 +1,40 @@
-<%= form_with(model: pair_request) do |form| %>
-  <% if pair_request.errors.any? %>
-    <div class="text-error">
-      <h2><%= pluralize(pair_request.errors.count, "error") %> prohibited this pair_request from being saved:</h2>
+<%= turbo_frame_tag "new_request_form" do %>
+  <%= form_with(model: pair_request, class: 'p-4 w-full mx-auto border-2 border-primary rounded-md flex flex-col') do |form| %>
+    <% if pair_request.errors.any? %>
+      <div class="alert alert-error flex flex-col items-start my-4">
+        <%= pluralize(pair_request.errors.count, "error") %> prohibited this pair   request from being saved:
+        <ul>
+          <% pair_request.errors.each do |error| %>
+            <li class='font-bold'>
+              <%= error.full_message %>
+            </li>
+          <% end %>
+        </ul>
+      </div>
```

Comment:
can we move this whole part into a generic view component called `FormErrorsComponent`?

Reply:
I think honestly this should all be shown in a flash. In the spirit of refactoring, I just moved and slightly improved the messaging, but I'd rather that in the `create.turbo_stream.erb` we render a flash message with either errors or success. I think this should be another ticket unto itself, might be another really good first issue/solo.

Reply:
#102 should address this issue